caf-ssl-dropwizard/src/main/java/com/github/cafapi/ssl/dropwizard/DropWizardSslBundle.java

```diff
@@ -21,16 +21,29 @@
 import io.dropwizard.core.setup.Environment;
 import io.dropwizard.jetty.ConnectorFactory;
 import io.dropwizard.jetty.HttpsConnectorFactory;
+
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
+import com.hpe.caf.secret.SecretUtil;
+
 enum DropWizardSslBundle implements ConfiguredBundle<Configuration>
 {
     INSTANCE;
 
     private static final String SSL_KEYSTORE_PATH = System.getenv("SSL_KEYSTORE_PATH");
     private static final String SSL_KEYSTORE = System.getenv("SSL_KEYSTORE");
-    private static final String SSL_KEYSTORE_PASSWORD = System.getenv("SSL_KEYSTORE_PASSWORD");
+    private static final String SSL_KEYSTORE_PASSWORD;
+
+    static {
+        try {
+            SSL_KEYSTORE_PASSWORD = SecretUtil.getSecret("SSL_KEYSTORE_PASSWORD");
```

Comment:
We should probably move this to normal runtime to avoid the possibility of this throwing an exception and the class not getting constructed at all.